runner/ansible/roles/1.3.1/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+
+- name: Set Test ID
+  set_fact:
+    test_id: '1.3.1'
+
+- name: "{{ test_id }}.check"
+  shell: 'grep SBD_PACEMAKER /etc/sysconfig/sbd'
```

Comment:
We should go back to the `replace` option, like in `1.1.1` test, as this is a file checking

Reply:
I think we need to go with line in file for this one. But will make that change for 1.3.1 and 1.3.2